server/command.go

```diff
@@ -230,6 +232,25 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 		}
 
 		return p.getCommandResponse(args, "Settings updated."), nil
+	case "build":
+		if len(parameters) < 2 {
+			return p.getCommandResponse(args, "Please specify both a repo and reference. Use `/gitlab help` for more usage information."), nil
+		}
+
+		repo := parameters[0]
+		branch := parameters[1]
+		user, err := p.getGitlabUserInfoByMattermostID(p.BotUserID)
+		if err != nil {
+			return p.getCommandResponse(args, "Encountered an error getting your GitLab profile."), nil
+		}
```

Comment:
Can we use `info` from L146 directly here?

Reply:
seems like it, I've made the changes